integration-libs/cdc/user-profile/register/cdc-register-component.service.ts

```diff
@@ -50,24 +43,26 @@ export class CDCRegisterComponentService extends RegisterComponentService {
    * @param user as UserSignUp
    */
   register(user: UserSignUp): Observable<User> {
+    if (!user.firstName || !user.lastName || !user.uid || !user.password) {
+      throw new Error(`The provided user is not valid: ${user}`);
```

Comment:
I believe, here we should `return throwError(new Error(...))`, because we're not in the scope of rxjs operator. I believe we want to emit an rxjs error instead of simply throw JS Error

Reply:
Great catch, fixed